Modules/_decimal/_decimal.c

```diff
@@ -548,6 +546,8 @@ getround(PyObject *v)
    initialized to new SignalDicts. Once a SignalDict is tied to
    a context, it cannot be deleted. */
 
+#define INVALID_SIGNALDICT_ERROR_MSG "invalid signal dict"
```

Comment:
How about
```c
const char* INVALID_SIGNALDICT_ERROR_MSG = "invalid signal dict";
```

Reply:
Looks good. Some experienced cpp programmers may prefer `const` ~. There is no harm to do it, changed!